agent/firewall.go

```diff
@@ -113,23 +117,26 @@ func (fw *Firewall) Init(netif NetIf) error {
 	// Allow ICMP, DHCP and SSH between host and instances.
 	hostAddr := fw.Agent.networkConfig.romanaGW
 	inputRules := []string{
-		fmt.Sprintf("-d %s/32 -p icmp -m icmp --icmp-type 0 -m state --state RELATED,ESTABLISHED", hostAddr),
-		fmt.Sprintf("-d %s/32 -p tcp -m tcp --sport 22", hostAddr),
 		"-d 255.255.255.255/32 -p udp -m udp --sport 68 --dport 67",
```

Comment:
These rules for DHCP traffic are only needed for OpenStack, right? Since they are here, does this mean that they will also be installed for K8S pods?